util/distro/distro.go

```diff
@@ -20,7 +24,7 @@ type DistributionResource struct {
 	TiFlash  string `json:"tiflash,omitempty"`
 }
 
-var defaultDistroRes = DistributionResource{
+var DefaultDistroRes = DistributionResource{
```

Comment:
There is no need to expose `DefaultDistroRes` to the external, as `R()` already provides it.

Reply:
Oh,  yep, I got it, because the default distro res won't be replaced by customized distro res when we only run the write_strings.go